Apps/Inc/CarState.h

```diff
@@ -58,6 +57,29 @@ typedef struct{
     State MotorConnectionErr;
 } error_code_t;
 
+/**
+ * Regen Brake Mode Enum
+ * 
+ * Different modes of the regenerative braking
+ * button
+ */
+typedef enum{OFF, RATE1, RATE2, RATE3} RegenMode;
```

Comment:
Since we already have an enum with an OFF defined, you'll have to rename this option to something else (maybe REGEN_OFF?).